src/Integration.Vsix/SonarLintTracker/IssueMarker.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Microsoft.VisualStudio.Text;
+using Sonarlint;
+
+namespace SonarLintTracker
+{
+    class IssueMarker
+    {
+        public readonly Issue Issue;
+        public readonly SnapshotSpan Span;
+
+        public IssueMarker(Issue issue, SnapshotSpan span)
```

Comment:
I know this is defensive programming but are you sure you don't want to check the arguments for null?